.gitlab/lassen.yml

```diff
@@ -20,38 +20,38 @@
 
 # Build MFEM
 build_mfem_ser_lassen:
-  extends: [.on_lassen]
+  extends: [.with_gcc_8_3_1, .on_lassen]
   needs: [setup]
   stage: lassen_build
   script:
     - mkdir -p ${BUILD_PATH}
     - cp -r ${CI_PROJECT_DIR} ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser
     - cd ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser
-    - lalloc 1 -W 5 -q pdebug make -j cuda
+    - lalloc 1 -W 5 -q pdebug make -j cuda CUDA_ARCH=sm_70
 
 build_mfem_debug_ser_lassen:
-  extends: [.on_lassen]
+  extends: [.with_gcc_8_3_1, .on_lassen]
   needs: [setup]
   stage: lassen_build
   script:
     - mkdir -p ${BUILD_PATH}
     - cp -r ${CI_PROJECT_DIR} ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser_debug
     - cd ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser_debug
-    - lalloc 1 -W 5 -q pdebug make -j cuda MFEM_DEBUG="YES"
+    - lalloc 1 -W 5 -q pdebug make -j cuda MFEM_DEBUG="YES" CUDA_ARCH=sm_70
 
 # Sanity check
 sanitycheck_mfem_ser_lassen:
-  extends: [.on_lassen]
+  extends: [.with_gcc_8_3_1, .on_lassen]
   stage: lassen_test
   needs: [build_mfem_ser_lassen]
   script:
     - cd ${BUILD_PATH}/${CI_PROJECT_NAME}_lassen_ser
-    - lalloc 1 -W 15 -q pdebug make -j test
+    - lalloc 1 -W 15 -q pdebug make -j test CUDA_ARCH=sm_70
```

Comment:
No need to set `CUDA_ARCH` in the testing steps -- it is remembered from the configuration+build steps.

Reply:
Done.